cms/static/sass/studio-main-v2-rtl.scss

```diff
@@ -1,14 +0,0 @@
-// ------------------------------
```

Comment:
@rlucioni We definitely don't want to remove this file (and most of the other scss changes). This work was reverted on master and then added back again. I don't know why the rebase is trying to remove it again, rather than realizing that this is essentially the same as the original revert.